### PR TITLE
Show "capture profile" button in the profile plugin even if there is no profile data.

### PR DESCRIPTION
**Pull request by qiuminxu**
* Motivation for features / changes
https://github.com/tensorflow/tensorboard/issues/2090
* Technical description of changes
Show the capture profile button when no data is found. Modify the FAQs to both TPU and CPU/GPU.
* Screenshots of UI changes
Before:
no_profile_data_found_old
After:
no_profile_data_found_gpu2

* Detailed steps to verify changes work correctly (as executed by you)
bazel run tensorboard:tensorboard -- logdir=.
http://localhost:6006/#profile
* Alternate designs / implementations considered

### CODE REVIEW

**Comment by qqfish** [comment]
Very cool. Thanks for making this change!

File: tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html [hunk 1]
```diff
@@ -93,51 +98,30 @@ <h3>Profiling isn’t supported in Colab yet.</h3>
...
-      rel="noopener"
-      target="_blank"
-    >Google Cloud TPU Troubleshooting and FAQ</a>
-    and consider filing an issue on GitHub.</p>
+
+    <paper-button class="profile-btn" raised on-tap="_openCaptureProfileDialog">Capture Profile</paper-button>
+
+    <p>If you have a model running on a Google Cloud TPU or a GPU, you may be
+        able to use the above button to capture a profile.
+    <p>If you're a TPU user, please use the TPU name option and you may want to check out the
```
Comment (wchargin) [inline tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html]: Please preserve closing `</p>` tags.

File: tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html [hunk 2]
```diff
@@ -93,51 +98,30 @@ <h3>Profiling isn’t supported in Colab yet.</h3>
...
+           <a href="https://cloud.google.com/tpu/docs/cloud-tpu-tools"
+               rel="noopener" target="_blank">tutorial</a>
+           on how to interpreting the profiling results.
+    <p>If you’re a GPU user, please use the IP address option and you may want to check out the
+    <a href="https://colab.research.google.com/github/tensorflow/tensorboard/blob/master/docs/r2/tensorboard_profiling_keras.ipynb" rel="noopenr" target="_blank">tutorial</a> on how to profile a Keras model on a GPU.
```
Comment (wchargin) [inline tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html]: `noopener`, not `noopenr`.

File: tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html [hunk 3]
```diff
@@ -45,6 +45,11 @@
 -->
 
 <dom-module id="tf-profile-dashboard">
+  <style>
+.profile-btn {
+  width: 160px;
```
Comment (wchargin) [inline tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html]: Why is this explicit width needed?

File: tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html [hunk 4]
```diff
@@ -93,51 +98,29 @@ <h3>Profiling isn’t supported in Colab yet.</h3>
...
-    <a
-      href="https://cloud.google.com/tpu/docs/troubleshooting"
-      rel="noopener"
-      target="_blank"
-    >Google Cloud TPU Troubleshooting and FAQ</a>
-    and consider filing an issue on GitHub.</p>
+
+    <paper-button class="profile-btn" raised on-tap="_openCaptureProfileDialog">Capture Profile</paper-button>
+
+    <p>If you have a model running on a Google Cloud TPU or a GPU, you may be
```
[the remark this answers is not in the document]
Comment (qiuminxu) [reply tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html]: Done. The profile actually works for both CPU and GPU as long as the profiler server is running.

File: tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html (hunk 1 above)
Comment (qiuminxu) [reply tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html]: Done.

File: tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html (hunk 2 above)
Comment (qiuminxu) [reply tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html]: Done. Thanks!

File: tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html [hunk 5]
```diff
@@ -93,51 +98,31 @@ <h3>Profiling isn’t supported in Colab yet.</h3>
...
-    <a
-      href="https://cloud.google.com/tpu/docs/troubleshooting"
-      rel="noopener"
-      target="_blank"
-    >Google Cloud TPU Troubleshooting and FAQ</a>
-    and consider filing an issue on GitHub.</p>
+
+    <paper-button class="profile-btn" raised on-tap="_openCaptureProfileDialog">Capture Profile</paper-button>
+
+    <p>If you have a model running on CPU, GPU or Google Cloud TPU, you may be
```
Comment (wchargin) [inline tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html]: nit: comma after “GPU”

Comment (qiuminxu) [reply tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html]: Done.

File: tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html (hunk 3 above)
Comment (qiuminxu) [reply tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html]: I was thinking to make the button in the side bar wider. It turns out having no effect, so I removed this.

**Pull request merged**